### PR TITLE
Updated Russian translation

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: avidemux/qt4/i18n/avidemux_ru.ts [hunk 1]
```diff
@@ -12626,22 +12627,22 @@ Please delete the idx2 file and reopen.</source>
...
         <location line="+1"/>
         <source>vaapi: Resize image using vaapi.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">vaapi: Изменить размер изображения с помощью vaapi.</translation>
     </message>
     <message>
         <location line="+134"/>
         <source>Width :</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Ширина :</translation>
```
Comment (eumagga0x2a) [inline avidemux/qt4/i18n/avidemux_ru.ts]: Such a space between a word and a punctuation mark is actually a rule in French. Other languages may follow their own orthography here, methinks.

File: avidemux/qt4/i18n/avidemux_ru.ts [hunk 2]
```diff
@@ -7166,32 +7167,32 @@ This message will not be displayed again.</source>
     <message>
         <location filename="../../../avidemux_plugins/ADM_videoFilters6/blackenBorder/qt4/blackenBorders.ui" line="+14"/>
         <source>Blacken Borders</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Затемнение границ</translation>
```
Comment (eumagga0x2a) [inline avidemux/qt4/i18n/avidemux_ru.ts]: Just wondering if "краёв" or "каймы" would be a more suitable translation of "borders" here as well as in some other places in the code. The word "граница" was never used to describe the border regions of an image in painting classes, 30+ years ago :-)

File: avidemux/qt4/i18n/avidemux_ru.ts (hunk 1 above)
Comment (TotalCaesar659) [inline avidemux/qt4/i18n/avidemux_ru.ts]: I thought it's a local rule, like for special symbols or something. You can correct it.

File: avidemux/qt4/i18n/avidemux_ru.ts (hunk 2 above)
Comment (TotalCaesar659) [inline avidemux/qt4/i18n/avidemux_ru.ts]: Yeah, "краёв" will be more acceptable. Fix it, please.

**Comment by eumagga0x2a** [comment]
Fine, I'll prepare and submit a small pull request just for these two details once the current PR gets merged. Please keep up the good work!

**Comment by TotalCaesar659** [comment]
Thanks, I'll do it when I have free time.

**Pull request merged**